examples/file-upload-react/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>File Upload</title>
+  </head>
+  <body>
+    <p>
+      File upload testing using
+      <a href="https://www.cypress.io/">Cypress Test Runner</a>.
+      See the source of this page (React component) and the Cypress integration
+      spec file.
+    </p>
+    <div id="app"></div>
+
+    <script src="https://fb.me/react-15.0.0.js"></script>
+    <script src="https://fb.me/react-dom-15.0.0.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/babel-core/5.8.34/browser.min.js"></script>
+    <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
+    <script type="text/babel">
+      var FileUploadApp = React.createClass({
+        render: function() {
+          return (
+            <div>
+              <h3>File upload</h3>
+              <input id="file1" type="file" onChange={this.fileUpload}/>
+            </div>
+          )
+        },
+
+        fileUpload (e) {
+          // either upload file directly from the event (from a test)
+          // or from the DOM file input list
+          const file = e.nativeEvent.testFile || e.nativeEvent.target.files[0]
+          const data = new FormData()
+          console.log('uploading file', file)
+          data.append("file", file, "file")
```

Comment:
I would probably update this last arg to be the actual filename of the uploading file, just to have this data be clearer.

Reply:
good suggestion